src/instance/app.cfg.example

```diff
@@ -28,6 +28,10 @@ UUID_API_URL = 'http://uuid-api:8080'
 # Works regardless of the trailing slash
 INGEST_API_URL = 'https://ingest-api.dev.hubmapconsortium.org'
 
+# URL for talking to Ontology API (default for DEV)
+# Works regardless of the trailing slash
+ONTOLOGY_API_URL = 'https://ontology-api.dev.hubmapconsortium.org'
```

Comment:
@ChuckKollar is this change relying on the UBKG ontology-api? I had a branch that introduced the UBKG stuff, not sure how this config got into your code.